node/bindnode/node.go

```diff
@@ -189,6 +189,9 @@ func unionMember(val reflect.Value) (int, reflect.Value) {
 	// The first non-nil field is a match.
 	for i := 0; i < val.NumField(); i++ {
 		elemVal := val.Field(i)
+		if elemVal.Kind() != reflect.Ptr {
+			panic("found unexpected non-pointer in a union field")
```

Comment:
Do you think it might be time to start trying to adopt a consistent prefix -- something like perhaps `"bindnode cannot match golang type to schema:"` -- to errors like this?"

It wouldn't be load-bearing, but I think it'd be nice to start aiming towards a slightly more structured world.

Reply:
Sure, will do. The panic trace should still point to `node/bindnode/node.go`, but if the panic gets recovered, then it's just the message string.